h/models/token.py

```diff
@@ -49,6 +49,12 @@ class Token(Base, mixins.Timestamps):
         nullable=True,
     )
 
+    user_id = sqlalchemy.Column(
+        sqlalchemy.Integer,
+        sqlalchemy.ForeignKey("user.id", ondelete="cascade"),
+        nullable=True,
```

Comment:
We most likely want "index=True" here otherwise the cascade will have to go over all rows sequentially. 80k rows won't make that fail but it won't be fast.

If we change it here we'd need to add the index creation to he migration.

Reply:
Added the `index=True` :+1: